oeps/oep-0014-proc-archive-repos.rst

```diff
@@ -93,14 +93,9 @@ Archive Steps
 3. Create an IT help ticket to update the description of the repository to
    begin with ``[ARCHIVED]``.
 
-4. Create a DevOps ticket to change the status of the repo to "non-monitored"
-   in Gemnasium, and to remove any integration or webhooks that DevOps may be
+4. Create a DevOps ticket to remove any integration or webhooks that they may be
```

Comment:
I think you can just remove this step until we add a new security integration.  As of right now, we're getting notifications via Github automatically so there is nothing for devops to maintain.  In the future I believe the tools team is considering looking at requires.io but that tooling doesn't exist right now.